### PR TITLE
PA-1445: Update tooltip design

### PR DESCRIPTION
**Pull request by agalbrai**
Updated tooltip design as per Bobbi's design

### CODE REVIEW

**Review by jdtoombs — APPROVED** [review]

**Review by Fosol — APPROVED** [review]
Nice

File: frontend/src/components/common/TooltipIcon.scss [hunk 1]
```diff
@@ -1,12 +1,25 @@
...
 .tooltip {
   white-space: pre-wrap;
   .tooltip-inner {
     text-align: left;
     max-width: 25rem;
+    background-color: $secondary-variant-color;
+    color: #ffffff;
+  }
+  .arrow:before {
+    border-top-color: $secondary-variant-color;
```
Comment (devinleighsmith) [inline frontend/src/components/common/TooltipIcon.scss:19]: Can you confirm that this works if the tooltip is drawn to the left, right or bottom of the icon?

**Comment by devinleighsmith** [comment]
This will fix the tooltip icon's tooltips, but won't affect the tooltip wrappers tooltips. Not sure if that was on purpose or you haven't come across the tooltip wrapper yet.

For the tooltip wrapper you'll either need to override some of the global styling in App.scss or create a new scss file for TooltipWrapper.tsx

**Comment by devinleighsmith** [comment]
I'm fine if you want to merge this and then fix the other tooltips in another PR @agalbrai

File: frontend/src/components/common/TooltipIcon.scss (hunk 1 above)
Comment (agalbrai) [reply frontend/src/components/common/TooltipIcon.scss:19]: it seems it doesn't, I'll fix that

Comment (agalbrai) [reply frontend/src/components/common/TooltipIcon.scss:19]: it doesn't, but from what I've seen all of our tooltips are from the top (the only one on the right is being removed)

**Comment by agalbrai** [comment]
@devinleighsmith from what I've seen all of the tooltipWrappers tooltips also changed with this

File: frontend/src/components/common/TooltipIcon.scss (hunk 1 above)
Comment (devinleighsmith) [reply frontend/src/components/common/TooltipIcon.scss:19]: @agalbrai

**Pull request merged**